src/Darnytsia.Creatio.Core/Features/Books/Commands/UpdateBookAuthorCommand.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System;
+
+namespace Darnytsia.Creatio.Core.Features.Books.Commands;
+
+public record UpdateBookAuthorCommand(Guid BookId) : IRequest;
```

Comment:
should rename to FillingBookAuthorsCommand

Reply:
changed